MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -967,15 +970,21 @@ public void procreate() {
 						tCal.add(GregorianCalendar.DAY_OF_YEAR, 40*7);
 						setDueDate(tCal);
 						campaign.addReport(getHyperlinkedName()+" has conceived");
+						if (campaign.getCampaignOptions().logConception()) {
+						    addLogEntry(campaign.getDate(), "Has conceived");
+				        }
 					}
 				} else if (getSpouse() != null) {
 					if (getSpouse().isActive() && !getSpouse().isDeployed() && getSpouse().getAge(campaign.getCalendar()) > 13) {
 						// 0.05% chance that this procreation attempt will create a child
-						if (Compute.randomInt(10000) < 2) {
+						if (Compute.randomInt(10000) < 200) {
```

Comment:
Not sure I follow, the 200 was a mistake in the first commit 28746ea
That was changed back to 2 in the second commit a5bb302

Should I redo the PR or is it fine with the fix from the second commit?